### PR TITLE
Feature/teacherdashboardstyling

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [x] Other (please describe): 
Styling for teacher side, specifically the landing page and the header
## What is the current behavior?

- No styling on header, or teacher dashboard.
## What is the new behavior?

- Styling
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/TeacherDashboard/TeacherDashboard.js [hunk 1]
```diff
@@ -21,24 +21,31 @@ const TeacherDashboard = ({ getTeacher, getStudents, getLessons }) => {
...
+    )
+  }
 
-const mapStateToProps = ({ setStudents, setLessons, setTeacher }) => ({
+  
+  const mapStateToProps = ({ setStudents, setLessons, setTeacher, isLoading }) => ({
  students: setStudents,
  lessons: setLessons,
- teacher: setTeacher
+ teacher: setTeacher,
```
Comment (KyleWong2510) [inline src/TeacherDashboard/TeacherDashboard.js:47]: Nice job remembering those

**Review by KyleWong2510 — APPROVED** [review]
Nice job on styling so far!  Looks great.

File: src/TeacherDashboard/TeacherDashboard.js [hunk 2]
```diff
@@ -21,24 +21,31 @@ const TeacherDashboard = ({ getTeacher, getStudents, getLessons }) => {
...
-        based on topics taught, and the app will auto generate a link that may be shared for all students.  Based on the results of the 
-        lesson, the teacher will be able to review results to see how well a student was able to digest that topic. 
-      </h3>
-    <div className="jokeWrapper">
-      <h4>Joke of the day!</h4>
-      <h3>What did the ocean say to the Pirate?</h3>
-      <h3>Nothing, it just waved!</h3>
-    </div>
+    {isLoading && <p>Loading</p>}
+    {!isLoading &&
```
Comment (Melizzo) [inline src/TeacherDashboard/TeacherDashboard.js:25]: Nice - thank you for adding in the loading text

File: src/TeacherHeader/TeacherHeader.test.js [hunk 3]
```diff
@@ -11,8 +11,8 @@ describe('TeacherHeader', () => {
     
     const title = getByText('gumberoo')
     const studentsBtn = getByText('Students')
-    const lessonsBtn = getByText('Lessons')
-    const createLessonBtn = getByText('Create a Lesson')
+    const lessonsBtn = getByText('Assessments')
+    const createLessonBtn = getByText('Create an Assessment')
```
Comment (Melizzo) [inline src/TeacherHeader/TeacherHeader.test.js:16]: Assessments - yes

**Review by Melizzo — APPROVED** [review]

File: src/TeacherDashboard/TeacherDashboard.js (hunk 1 above)
Comment (Melizzo) [reply src/TeacherDashboard/TeacherDashboard.js:47]: I don't see what is different for the teacher? 😬

**Pull request merged**